src/nupic/os/Env.cpp

```diff
@@ -126,6 +126,8 @@ char ** Env::environ_ = nullptr;
 
 #if defined(NTA_OS_DARWIN)
 #include <crt_externs.h>
+#elif defined(NTA_OS_WINDOWS)
+_CRTIMP extern char **_environ;
```

Comment:
> Did you just set this after seeing some failure on Windows? It would be better to have a portable library for this functionality.

It was not me that implemented this, I'll check whether Windows build pass without this code.